pom.xml

```diff
@@ -14,7 +14,7 @@
   <name>Quarkus - MicroProfile - Parent</name>
 
   <properties>
-    <quarkus.version>3.12.2</quarkus.version>
+    <quarkus.version>999-SNAPSHOT</quarkus.version>
```

Comment:
Looks like this version doesn't exit and the build is failing. Can you fix, would be usefull so we can use microprofile 7 with only including 1 dependency.

Reply:
It is on purpose, so I can try with the latest Quarkus Snapshot (the PR is still draft). The latest Quarkus release does not have some of the required pieces to move to MP 7.

Reply:
Thanks for the help. Most of the work is done. We are still missing some integration pieces between Fault Tolerance and OpenTelemetry. I'm unsure if this will be ready in this Quarkus release or later.

We need the Quarkus 3.17 release which is scheduled here: 
https://github.com/quarkusio/quarkus/wiki/Release-Planning